### DISCUSSION TITLE
如何预览 .mhtml的文件？

### DISCUSSION THREAD

**Discussion by leiwcom** [AlistGo/alist#7213] [Ideas]
大佬们，如题，mhtml的文件如何实现在AList里直接预览？
AList会在近期更新支持mhtml的预览吗？？？？？？

**Comment by anwen-anyi** [comment]
自己写个可以预览解析 `HTML`的小插件（类似PDF.js这样的）然后配置一下
https://alist.nn.ci/zh/config/preview.html#iframe-%E9%A2%84%E8%A7%88
